### DISCUSSION TITLE
RUM spouštěné na localhostu

### DISCUSSION THREAD

**Discussion by lucienevludova-direct** [canarytrace/documentation#215] [RUM]
Ahoj, 

chtěli bychom se zeptat, zda se už někdo nesetkal s tím, že na localhostu nefunguje RUMko? Stav je takový, že se po spuštění projektu a otevření webové stránky na localhostu stáhne rum skript, ale poté se již v pravidelných intervalech nespouští dál. Kromě stažení skriptu se nikdy nespustí, zkoušeli jsme různé OS, různé prohlížeče, vždy se stejným výsledkem. 
Pokud stejné změny, jako ty, které jsme chtěli zkusit na localhostu, nasadíme na vývojové prostředí, tak již všechno funguje, Hero elementy se propisují, Kibana se plní daty.
Nějaké nápady, čím by to mohlo být, nebo co bychom mohli pro zprovoznění zkusit udělat? Prodlužuje se nám tím čekání na vyzkoušení změn, protože při nasazení samozřejmě musí proběhnout celý build, i když se jedná jen o malou změnu. 

Děkuju a přeji pěkný den
Lucka Nevludová

**Comment by rdpanek** [comment]
Ahoj, 
jde o CORS a defaultním chování webového browseru. Předpokládám že `same-origin-policy` a pokud by jsi to chtěla jinak, musela by jsi nastartovat browser s jiným nastavením.

Pokud se potřebujete testovat něco co půjde do produkce, tak se stále může použít nasazené RUMko na serveru a pohrávat si se změnami díky override ne?

Díky Dan
